nestkernel/connection_manager.cpp

```diff
@@ -376,29 +385,42 @@ nest::ConnectionManager::get_user_set_delay_extrema() const
   return user_set_delay_extrema;
 }
 
-nest::ConnBuilder*
+nest::BipartiteConnBuilder*
 nest::ConnectionManager::get_conn_builder( const std::string& name,
   NodeCollectionPTR sources,
   NodeCollectionPTR targets,
+  ThirdOutBuilder* third_out,
```

Comment:
sorry, I'm a bit slow, could you explain why the bipartite connection builder needs a pointer to a tri-partite builder too?

Reply:
The bipartite builder connects primary sources to primary targets. For each primary source-target connection created, it must call `third_out`, so that builder can check if there shall also be a third-factor connection.